theorems/homotopy/EilenbergMacLane1.agda

```diff
@@ -72,31 +74,460 @@ module homotopy.EilenbergMacLane1 {i} (G : Group i) where
       transport (λ x → G.ident == x [ fst ∘ Codes ↓ emloop g ])
                  (G.unit-l g) (↓-Codes-loop g G.ident)
 
+
+    -- TODO: move this elsewhere
+    foo : ∀ {i j k} {A : Type i} {B : A → Type j} {C : A → Type k}
+      {x x' : A} {p₁ p₂ : x == x'} (q : p₁ == p₂)
+      (u : B x → C x) (u' : B x' → C x')
+      (s : transport C p₁ ∘ u == u' ∘ transport B p₁)
+      (t : transport C p₂ ∘ u == u' ∘ transport B p₂)
+      → (s ∙' ap (λ p → u' ∘ transport B p) q) == (ap (λ p → transport C p ∘ u) q ∙ t)
+      -- → s == t [ (λ p → transport C p ∘ u == u' ∘ transport B p) ↓ q ]
+      → ↓-→-from-transp s == ↓-→-from-transp t [ (λ p → u == u' [ (λ y → B y → C y) ↓ p ]) ↓ q ]
+    foo q u u' s t h = ap↓ ↓-→-from-transp (↓-='-in h)
```

Comment:
Hmm... maybe we can make this local, at least?

Reply:
`foo` was only used in one place. I've inlined it.